pkg/proto/datadog/remoteconfig/remoteconfig.proto

```diff
@@ -0,0 +1,101 @@
+syntax = "proto3";
+
+package datadog.config;
+
+option go_package = "pkg/proto/pbgo";  // golang
+
+// Backend definitions
+
+message ConfigMetas {
+  repeated TopMeta roots = 1;
+  TopMeta timestamp = 2;
+  TopMeta snapshot = 3;
+  TopMeta topTargets = 4;
+  repeated DelegatedMeta delegatedTargets = 5;
+}
+
+message DirectorMetas {
+  repeated TopMeta roots = 1;
+  TopMeta timestamp = 2;
+  TopMeta snapshot = 3;
+  TopMeta targets = 4;
+}
+
+message DelegatedMeta {
+  uint64 version = 1;
+  string role = 2;
+  bytes raw = 3;
+}
+
+message TopMeta {
+  uint64 version = 1;
+  bytes raw = 2;
+}
+
+message File {
+  string path = 1;
+  bytes raw = 2;
+}
+
+// Backend queries
+
+message LatestConfigsRequest {
+  string hostname = 1;
+  string agentVersion = 2;
+  // timestamp and snapshot versions move in tandem so they are the same.
+  uint64 current_config_snapshot_version = 3;
+  uint64 current_config_root_version = 9;
+  uint64 current_director_root_version = 8;
+  repeated string products = 4;
+  repeated string new_products = 5;
+  repeated Client active_clients = 6;
+}
+
+message LatestConfigsResponse {
+  ConfigMetas config_metas = 1;
+  DirectorMetas director_metas = 2;
+  repeated File target_files = 3;
+}
+
+// Client defintions
+
+message Client {
+  ClientState state = 1;
+  repeated string products = 2;
+  string id = 3;
+  string name = 4;
+  string version = 5;
+  bool is_tracer = 6;
+  TracerClient tracer_client = 7;
+}
+
+message TracerClient {
+  string service = 1;
+  string env = 2;
+  string version = 3;
```

Comment:
This misses the tracer tag RuntimeID used to identify a tracer (in config.proto). I guess you wanted to merge with client.ID. I think it's better to keep it an explicit tracer tag. Probably client.ID can be removed as dupe of client.Name

Other tags you'll want for tracking and debugging tracers:
- appVersion (expliciting which version)
- language
- tracerVersion

Reply:
Added the suggested ones but I can't remove client.ID as it's used to dedupe clients in the active client logic. Technically I could write an ID() method but this sounds error prone